src/database/models/userModel.ts

```diff
@@ -25,6 +53,45 @@ class User extends Model<UserAttributes> implements UserAttributes {
   public password!: string
 
   public userRole!: string
+
+  public gender!: string
+
+  public birthdate!: string
+
+  public phoneNumber!: string
+
+  public preferredLanguage!: string
+
+  public preferredCurrency!: string
+
+  public homeAddress!: string
+
+  public billingAddress!: string
+
+  public profileImageUrl?: string
+
+  public readonly createdAt!: Date
+
+  public readonly updatedAt!: Date
+
```

Comment:
```suggestion

```
it would be better if you had user profile as new table and add one to one relationship

Reply:
Done